test/chat_api/conversations_test.exs

```diff
@@ -709,6 +709,51 @@ defmodule ChatApi.ConversationsTest do
     end
   end
 
+  describe "list_conversations_by_account_paginated/1" do
+    test "sorts the conversations by most recent activity" do
+      account = insert(:account)
+      filter = %{"limit" => 5}
+      base_last_activity_at = DateTime.from_naive!(~N[2020-12-01 00:00:00], "Etc/UTC")
+
+      [third_batch, second_batch, first_batch] =
+        15..1
```

Comment:
maybe i'm reading this incorrectly but just curious why not `1..15` and then use `first_batch, second_batch, third_batch` above?

Reply:
I used 15...1 because I want the results to have high to low `last_activity_value`. That way the first page will be the ones with the most recent activity, then the second page will have results with 2nd most recent activity

I guess I can change the name from `third_batch, second_batch, first_batch` to `first_batch, second_batch, third_batch`

Reply:
ah i see -- yeah maybe reverse the naming? that would be a little less confusing (otherwise it just feels like everything is flipped)